### PR TITLE
akefile.unix, tool/mkbinheader.py: save uncompressed binary at output

### PR DESCRIPTION
**Pull request by sunghan-chang**
For debugging purpose, leave uncompressed binary even CONFIG_COMPRESSED_BINARY is enabled.

Signed-off-by: sunghan-chang <sh924.chang@samsung.com>

### CODE REVIEW

File: loadable_apps/loadable.mk [hunk 1]
```diff
@@ -79,6 +79,7 @@ ifeq ($(CONFIG_ELF_EXCLUDE_SYMBOLS),y)
 	$(Q) $(OBJCOPY) --remove-section .comment $(USER_BIN_DIR)/$(BIN)
 	$(Q) $(STRIP) -g $(USER_BIN_DIR)/$(BIN) -o $(USER_BIN_DIR)/$(BIN)
 endif
+	$(Q) cp $(USER_BIN_DIR)/$(BIN) $(USER_BIN_DIR)/$(BIN).uncomp
```
Comment (jeongchanKim) [inline loadable_apps/loadable.mk:82]: How about adding condition for `CONFIG_SUPPORT_COMMON_BINARY`?

**Review by jeongchanKim — COMMENTED** [review]
Please fix the typo in commit title.
`lodable_apps` -> `loadable_apps`.

File: loadable_apps/loadable.mk (hunk 1 above)
Comment (sunghan-chang) [reply loadable_apps/loadable.mk:82]: It's not the content of COMMON_BINARY. It's for COMPRESSED_BINARY. Let me move to work only with it.

**Comment by sunghan-chang** [comment]
> Please fix the typo in commit title.
lodable_apps -> loadable_apps.

Ok. will do.

File: loadable_apps/loadable.mk (hunk 1 above)
Comment (jeongchanKim) [reply loadable_apps/loadable.mk:82]: Oh sorry. Yes, I mean for compress binary.

**Review by davidfather — DISMISSED** [review]
Looks good to me.

File: loadable_apps/loadable.mk (hunk 1 above)
Comment (sunghan-chang) [reply loadable_apps/loadable.mk]: I applied this to work with COMPRESSED_BINARY enabled.

**Review by sangwon03 — APPROVED** [review]
LGTM

**Pull request merged**